### DISCUSSION TITLE
Stream Listener

### DISCUSSION THREAD

**Discussion by rwanjohi** [tweepy/tweepy#1679] [Questions] [answered]
Can't load StreamListener anymore.Any updates? new location?
 
ImportError: cannot import name 'StreamListener' from 'tweepy.streaming'

**Comment by Harmon758** [comment, marked as answer]
`StreamListener` has been merged with [`Stream`](https://tweepy.readthedocs.io/en/v4.0.1/stream.html#tweepy.Stream) in [Tweepy v4.0.0](https://github.com/tweepy/tweepy/releases/tag/v4.0.0).
